.github/labeler.yml

```diff
@@ -0,0 +1,29 @@
+documentation:
```

Comment:
we do already have the same label coming from our github bot. Should we do this from both the places or keep one alive?

Reply:
The bot attaches label as per the PR title, I think it's fine to keep it at both places.